README.md

```diff
@@ -37,8 +37,14 @@ npm i -g azure-functions-core-tools@3 --unsafe-perm true
 To install with chocolatey:
 
 ```bash
+**v2**
```

Comment:
I think you might want to put "\*\*v2\*\*" above bash, otherwise it renders the "v2" inside the command box thing.

I recommend changing 
```
```bash
**v2**
```

to 

```
**v2**
```bash
```

Reply:
updated syntax